tools/taskcluster/upload-coverage-report.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Similarly, as a reviewer, it'd be helpful to have a comment where the pattern for these changes and this file came from if applicable (it looks like it's following what a-c did).